tests/PasswordStrengthTest.php

```diff
@@ -26,6 +28,12 @@
  */
 class PasswordStrengthTest extends SapphireTest
 {
+    protected function setUp(): void
+    {
+        parent::setUp();
+        Injector::inst()->registerService(new PasswordValidator(), PasswordValidator::class);
```

Comment:
I think the problem is coming from this line in SapphireTest: https://github.com/silverstripe/silverstripe-framework/blob/c1427ff9c4c7711336eebfbbbf0f48bded509507/src/Dev/SapphireTest.php#L283-L285

When I comment out the line locally, the test passes without issue.

Not sure why SapphireTest wants to nerf the password validator ... my guess is that it gets in the way of some other testing logic. In this context, it seems reasonable to manually re-register the validator.

I think this approach is better because it will use the original validator rather than instantiating a new one.

```suggestion
        // Out of the box, SapphireTest wants to deregister the PasswordValidator. However, we need for these test to work.
        $validator = Member::password_validator();
        parent::setUp();
        Member::set_password_validator($validator);
```

Reply:
Updated